### PR TITLE
Daria P. first update

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: students.txt [hunk 1]
```diff
@@ -12,7 +12,9 @@ List of students
 5. Dima P.
 6. Vladimir S.
 7. Daniil I.
-8. Daria P. 
+8. Daria Perepichko
+I like drawing, dancing, traveling, reading and active sports.
```
Comment (vit-lebediev) [inline students.txt]: Good, but can you use a list to name all your hobbies?
Use this as a template:
- my hobby 1
- my hobby 2
- ...

**Review by vit-lebediev — CHANGES_REQUESTED** [review]
Overall looks good, small changes

**Comment by ghost** [comment]
add second update

**Review by vit-lebediev — APPROVED** [review]

**Comment by vit-lebediev** [comment]
:shipit:

**Pull request merged**